dashboard/src/components/Layout/Clarity.tsx

```diff
@@ -29,11 +29,11 @@ import {
   timesIcon,
   trashIcon,
   uploadCloudIcon,
-} from "@clr/core/icon";
-import "@clr/core/icon/register.js";
+} from "@cds/core/icon";
+import "@cds/core/icon/register.js";
 import * as React from "react";
 
-import "@clr/ui/clr-ui.min.css";
+import "@clr/ui/clr-ui.min.css"; // light clarity UI theme
```

Comment:
shouldn't we use just one place to import clarity css files? (either here or in the index file, that btw is failing to import the files).

Reply:
Yep, this is exactly what we are doing in https://github.com/kubeapps/kubeapps/pull/2450
I just left unchanged this file (apart from clarifying it is the light theme)